packages/sdk-js/src/types/growthbook.ts

```diff
@@ -332,6 +334,7 @@ export type GlobalContext = {
 export type UserContext = {
   enabled?: boolean;
   qaMode?: boolean;
+  enableDevMode?: boolean;
```

Comment:
presuming that devMode should be user context rather than a property of the user-scoped SDK...

Reply:
We should also set this in `GrowthBook._getUserContext` for consistency.